ocs_ci/helpers/performance_lib.py

```diff
@@ -86,3 +89,123 @@ def run_command(cmd, timeout=600, out_format="string", **kwargs):
         output.pop()  # remove last empty element from the list
     return output
+
+
+def run_oc_command(cmd, namespace=constants.OPENSHIFT_STORAGE_NAMESPACE):
+    """
+    Running an 'oc' command
+    This function is needed in Performance tests in order to be able to run a ceparate command within the test
+    without creating additional objects which increases memory consumed by the test.
+
+    Args:
+        cmd (str): the command to run
+        namespace (str): the namespace where to run the command
+
+    Returns:
+        list : the results of the command as list of lines
+
+    """
+
+    cluster_dir_kubeconfig = os.path.join(
+        config.ENV_DATA["cluster_path"], config.RUN.get("kubeconfig_location")
+    )
+    if os.getenv("KUBECONFIG"):
+        kubeconfig = os.getenv("KUBECONFIG")
+    elif os.path.exists(cluster_dir_kubeconfig):
+        kubeconfig = cluster_dir_kubeconfig
+    else:
+        kubeconfig = None
+
+    command = f"oc --kubeconfig {kubeconfig} -n {namespace} {cmd}"
+    return run_command(command, out_format="list")
+
+
+def get_log_names(interface):
+    """
+    Finds names for log files pods in which logs for clone creation are located
+    For CephFS: 2 pods that start with "csi-cephfsplugin-provisioner" prefix
+    For RBD: 2 pods that start with "csi-rbdplugin-provisioner" prefix
+
+    Args:
+        interface (str) : an interface (RBD or CephFS) to run on
+
+    Returns:
+        log names (list) : names of the log files relevant for searching in
+
+    """
+    log_names = []
+
+    pods = run_oc_command(cmd="get pod", namespace="openshift-storage")
+
+    if "Error in command" in pods:
+        raise Exception("Can not get csi controller pod")
+
+    provisioning_name = "csi-cephfsplugin-provisioner"
+    if interface == constants.CEPHBLOCKPOOL:
+        provisioning_name = "csi-rbdplugin-provisioner"
+
+    for line in pods:
+        if provisioning_name in line:
+            log_names.append(line.split()[0])
+
+    logger.info(f"The logs pods are : {log_names}")
+    return log_names
+
+
+def measure_pvc_creation_time(interface, pvc_name, start_time):
+    """
+
+    Measure PVC creation time, provided pvc name and time after which the PVC was created
+
+    Args:
+        interface : an interface (RBD or CephFS) to run on
+        pvc_name : Name of the pvc for which we measure the time
+        start_time: Time from which and on to search the relevant logs
+
+    Returns:
+        creation time for PVC in seconds
+
+    """
+    log_names = get_log_names(interface)
+    logs = []
+    for l in log_names:
+        logs.append(
+            run_oc_command(
+                f"logs {l} -c csi-provisioner --since-time={start_time}",
+                "openshift-storage",
+            )
+        )
+
+    format = "%H:%M:%S.%f"
+
+    st = None
+    et = None
+    # look for start time and end time of clone creation. The start/end line may appear in log several times
```

Comment:
The comment mentions 'clone creation'. I believe this is a generic function. maybe it was copied from somewhere? if so, please update the comment to remove info related to clone.

Reply:
@keesturam Done, thank you!